### PR TITLE
Update airbase to 149 and airlift to 240 and enable more checks

### PR DESCRIPTION
**Pull request by wendigo**
Closes https://github.com/trinodb/trino-gateway/issues/184

### CODE REVIEW

File: gateway-ha/pom.xml [hunk 1]
```diff
@@ -137,18 +193,19 @@
...
+                </exclusion>
+            </exclusions>
         </dependency>
 
         <dependency>
             <groupId>io.dropwizard</groupId>
             <artifactId>dropwizard-views</artifactId>
         </dependency>
 
-        <dependency>
```
Comment (mosabua) [inline gateway-ha/pom.xml:147]: We found that this is needed for the UI to work .. we might have to experiment with a different scope or somehow figure out how the dependency analyse run can be satisfied.

File: gateway-ha/pom.xml [hunk 2]
```diff
@@ -280,23 +326,29 @@
             <version>${dep.jeasy.version}</version>
         </dependency>
 
-        <dependency>
```
Comment (mosabua) [inline gateway-ha/pom.xml:283]: This dep was added to fix an issue as well .. we need to verify this does not come up again.

File: gateway-ha/pom.xml (hunk 1 above)
Comment (mosabua) [reply gateway-ha/pom.xml:147]: See https://github.com/trinodb/trino-gateway/issues/213

File: gateway-ha/pom.xml [hunk 3]
```diff
@@ -312,6 +364,20 @@
...
+            <artifactId>ehcache</artifactId>
+            <version>${dep.ehcache.version}</version>
+            <scope>runtime</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.mvel</groupId>
+            <artifactId>mvel2</artifactId>
+            <version>2.5.1.Final</version>
+            <scope>runtime</scope>
```
Comment (mosabua) [inline gateway-ha/pom.xml:378]: oh.. cool .. I see

**Review by mosabua — DISMISSED** [review]

File: gateway-ha/pom.xml (hunk 2 above)
Comment (mosabua) [reply gateway-ha/pom.xml:283]: I see.. you just moved it down to runtime scope.

Comment (wendigo) [reply gateway-ha/pom.xml:283]: Yep

**Comment by wendigo** [comment]
I've tested UI locally. It renders. No errors in the logs.

**Comment by mosabua** [comment]
So good.. you rock @wendigo ! I was gonna get to it eventually but you were faster.. nice!

**Review by mosabua — APPROVED** [review]
Awesome .. thanks for all the clean up. The final jar with dependencies probably shrunk quite a bit more again. Love it.

**Pull request merged**